brigade/templates/resources_videos.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+{% block title %}Videos{% endblock %}
+{% block description %}Recordings from events within the Brigade Network{% endblock %}
+{% block page_id %}videos{% endblock %}
+
+{# Conditionals based on what type of page is rendering #}
+{% if topic %}
+{% else %}
+{% endif %}
+
+{% block content %}
+<header id="intro" class="page-header slab slab-dark-blue">
+  <div class="grid-box">
+    <h1 class="title">Videos</h1>
+    <div class="subtitle">Recordings from events within the Brigade Network</div>
+  </div>
+</header>
+
+{% if not topic %}
+<section class="slab slab--compact">
+  <div class="grid-box">
+    <div class="width-one-half">
```

Comment:
Minor design nitpick: consider making this `<div class="width-two-thirds">` for visual consistency with the three column grid layout on the page.
Eg:
![image](https://user-images.githubusercontent.com/8628090/40394120-00126c12-5dd7-11e8-9af8-bca5c4236dca.png)

Reply:
Done. Looks good.